internal/iam/command_pool_create.go

```diff
@@ -26,6 +26,7 @@ func (c *poolCommand) newCreateCommand() *cobra.Command {
 	pcmd.AddProviderFlag(cmd, c.AuthenticatedCLICommand)
 	cmd.Flags().String("identity-claim", "", "Claim specifying the external identity using this identity pool.")
 	cmd.Flags().String("description", "", "Description of the identity pool.")
+	pcmd.AddResourceOwnerFlag(cmd, c.AuthenticatedCLICommand)
```

Comment:
IMO, this function should not be publicly exposed since it's only used within this package. Could you add it to this package and make sure it's private?